src/platform/packages/shared/controls/controls-schemas/src/controls_group_schema.ts

```diff
@@ -45,22 +45,30 @@ export const chainingSchema = schema.oneOf(
 );
 
 export const ignoreParentSettingsSchema = schema.object({
-  ignoreFilters: schema.boolean({
-    meta: { description: 'Ignore global filters in controls.' },
-    defaultValue: DEFAULT_IGNORE_PARENT_SETTINGS.ignoreFilters,
-  }),
-  ignoreQuery: schema.boolean({
-    meta: { description: 'Ignore the global query bar in controls.' },
-    defaultValue: DEFAULT_IGNORE_PARENT_SETTINGS.ignoreQuery,
-  }),
-  ignoreTimerange: schema.boolean({
-    meta: { description: 'Ignore the global time range in controls.' },
-    defaultValue: DEFAULT_IGNORE_PARENT_SETTINGS.ignoreTimerange,
-  }),
-  ignoreValidations: schema.boolean({
-    meta: { description: 'Ignore validations in controls.' },
-    defaultValue: DEFAULT_IGNORE_PARENT_SETTINGS.ignoreValidations,
-  }),
+  ignoreFilters: schema.maybe(
```

Comment:
nit: As these settings aren't exposed in the UI anymore, it might be prudent for us to remove them from the schema, to potentially make room for a different format of these settings in the future.